pkg/ecctl/init.go

```diff
@@ -48,6 +48,12 @@ const (
 	jsonFormatChoice
 )
 
+const (
+	_ = iota
+	essInfraChoice
+	eceInfraChoice
+)
+
 const (
 	disclaimer      = "Welcome to the Elastic Cloud CLI! This command will guide you through authenticating and setting some default values.\n\n"
```

Comment:
for ecctl we use the phrase "Elastic Cloud Control (ecctl)" rather Elastic Cloud CLI